src/view/HomePage.java

```diff
@@ -272,12 +284,34 @@ private void btnBuscarHospedeActionPerformed(java.awt.event.ActionEvent evt) {//
     }//GEN-LAST:event_btnBuscarHospedeActionPerformed
 
     private void btnBuscarUsuarioActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnBuscarUsuarioActionPerformed
-        // TODO add your handling code here:
+        //Listar Usuarios
+        String nome = txtUsernameBusca.getText();
+        MySQL conn = new MySQL();
+        conn.conectaBanco();
+        try {
+            ControllerListarUsuarios usuario = new ControllerListarUsuarios();
```

Comment:
Como dito no comentário acima, basicamente essa linha vai para o início da classe. Vai ficar assim:
`ControllerListarUsuarios controllerListarUsuarios;`

Aí dentro do construtor da homePage vc instancia ele:
`controllerLIstarUsuarios = new ControllerListarUsuarios()`